getting-started/setup-building.rst

```diff
@@ -58,6 +58,26 @@ itself. Git is easily available for all common operating systems.
   ``git push``, or ``git fetch``.  On Windows, you should also
   :ref:`enable autocrlf <autocrlf>`.
 
+.. _install-pre-commit:
+
+Install pre-commit
+------------------
+
+To make sure your code is formatted correctly, we recommend installing
+`pre-commit <https://pre-commit.com>`_, and set it up as a Git hook.
```

Comment:
Hm, I think this is more correct. It would be nice to have a native speaker verify it, though.

```suggestion
To make sure your code is formatted correctly, we recommend installing
`pre-commit <https://pre-commit.com>`_, and setting it up as a Git hook.
```

Reply:
I just checked with www.deepl.com, looks good.